### ISSUE TITLE
[bug]: Advanced filter combobox ring is not fully visible

### ISSUE THREAD

**Issue by HZooly** [sadmann7/shadcn-table#648]
### Describe the bug

On keyboard navigation or after clicking on a combobox item, the focus ring is not visible on top/bottom. Shadow is not visible too.

### How to reproduce

- Toggle "Advanced filters"
- Keyboard focus "Title"

### Link to reproduction

https://table.sadmn.com/?status=todo&priority=medium&flags=advancedFilter&filters=%5B%7B%22id%22:%22title%22,%22value%22:%22%22,%22type%22:%22text%22,%22operator%22:%22iLike%22%7D%5D

### Additional information

<img width="606" alt="image" src="https://github.com/user-attachments/assets/9dcfb28d-ac1c-4eb9-a9b9-232d02d2dd6f">
